src/Cli/dotnet/commands/dotnet-test/TestApplication.cs

```diff
@@ -162,5 +262,15 @@ internal void OnSessionEvent(TestSessionEvent sessionEvent)
         {
             SessionEventReceived?.Invoke(this, new SessionEventArgs { SessionEvent = sessionEvent });
         }
+
+        internal void OnCreated()
```

Comment:
I don't think nobody can subscribe to this one...in the code we create and fire it there's not time for anyone to subscribe or? Maybe is not needed.

Reply:
Never mind me, you're enqueuing before the invoke.